### ISSUE TITLE
[Bug] Bot Fails to return quote when sending multiple messages in quick succession on slack

### ISSUE THREAD

**Issue by JoeTot** [obsantos/MarketBot#4]
## Overview
When requesting a quote via a channel in slack if the user sends the quote symbol (i.e. $NVDA) followed quickly by another message ('foo') the bot will fail to return the quote. 

Pinging the bot again after with just the symbol will return the quote as normal.

## Steps to Reproduce
- Send valid quote symbol
- Quickly send a second message
- Observe no quote is returned
